docs/source/release_notes/releasev0.8.0.rst

```diff
@@ -0,0 +1,133 @@
+.. _releasev0.8.0:
+
+==============================
+ Release notes v0.8.0
+==============================
+
+Quick Overview
+--------------
+
+* Physically Based Rendering (PBR) module added.
```

Comment:
New Principled BRDF functionality added. BRDF stands for bidirectional reflectance distribution function.

Reply:
For line 10 we should also say. New Physically Based Rendering (PBR) added. It includes anisotropic rotation among other features.